extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson-common/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/ResteasyReactiveCommonJacksonProcessor.java

```diff
@@ -54,15 +58,20 @@ void handleJsonAnnotations(Optional<ResourceScanningResultBuildItem> resourceSca
                                         "Class '" + biFunctionClassInfo.name() + "' must contain a no-args constructor");
                             }
                         }
-                        reflectiveClass.produce(
+                        reflectiveClassProducer.produce(
                                 new ReflectiveClassBuildItem(true, false, false, biFunctionType.name().toString()));
                     }
                 }
             }
         }
         if (!classesNeedingReflectionOnMethods.isEmpty()) {
-            reflectiveClass.produce(
+            reflectiveClassProducer.produce(
                     new ReflectiveClassBuildItem(true, false, classesNeedingReflectionOnMethods.toArray(new String[0])));
         }
+        if (!jacksonFeatures.isEmpty()) {
```

Comment:
just nitpicking, but this `if` seems redundant?

Reply:
Yeah, you are absolutely right, it got leftover from a previous iteration :(